spaceship/lib/spaceship/tunes/tunes_client.rb

```diff
@@ -155,7 +154,16 @@ def send_login_request(user, password)
         return response
       else
         if response["Location"] == "/auth" # redirect to 2 step auth page
-          raise "spaceship / fastlane doesn't support 2 step enabled accounts yet. Please temporary disable 2 step verification until spaceship was updated."
+          if File.exist?(persistent_cookie_path)
+            @cookie.load(persistent_cookie_path)
+            if self.team_id.to_s.length > 0
+              return true
+            else
+              File.delete(persistent_cookie_path) # session is already invalid
+            end
+          end
```

Comment:
I know there is a request to add the current device to the trusted devices. I expected this to update the cookie expiry date, but it seems like it doesn't. I'll have to further investigate